.github/workflows/rust.yml

```diff
@@ -30,7 +30,16 @@ jobs:
     steps:
     # This repo is unstable and unnecessary: https://github.com/microsoft/linux-package-repositories/issues/34
     - name: Disable packages.microsoft.com repo
+      if: ${{ startsWith(matrix.os, 'ubuntu') }}
       run: sudo rm -f /etc/apt/sources.list.d/microsoft-prod.list
+    # https://github.com/oxidecomputer/omicron/issues/4920
+    - name: Pin libxmlsec1 to 1.3.2
+      if: ${{ startsWith(matrix.os, 'macos') }}
+      run: |
+        curl -fLOsS --retry 5 https://raw.githubusercontent.com/Homebrew/homebrew-core/081149b0d2720c2759b6ac8253e33b27f6d6c1cd/Formula/lib/libxmlsec1.rb
+        brew install ./libxmlsec1.rb
+        brew pin libxmlsec1
+        rm -f libxmlsec1.rb
```

Comment:
Don't need the `-f` here because this is just a regular file that's getting removed.